test/test_geant4.jl

```diff
@@ -8,7 +8,7 @@ using RadiationDetectorSignals
 using TypedTables
 using Unitful
 
-T = Float32
+T = Float64
```

Comment:
Currently, `Polycone` has no implementation of `csgtol` which will cause some points that are "inside" the `Polycone` in `Float64` to be "outside" in `Float32`

Reply:
--> hopefully resolved in 1978adb